banners/mobile/messages.ts

```diff
@@ -22,7 +22,7 @@ const messages: TranslationMessages = {
 	'address-type-notice-none': 'Sie verzichten sowohl auf eine Spendenquittung als auch auf eine Bestätigung ' +
 		'per E-Mail. Sie erhalten von uns keine Information, wenn Wikipedia wieder Hilfe braucht.',
 	'soft-close-prompt': 'Wikipedia später unterstützen?',
-	'use-of-funds-link': 'Was Ihre Spende bewirkt',
+	'use-of-funds-link': 'Deine Spende wirkt',
```

Comment:
This will change the link in both banners to informal (while the CTRL stay formal entirely). The easiest way to recitify this will probably be a `messages_var.ts` included in the entry point

Reply:
Done.